Dockerfile

```diff
@@ -1,22 +1,48 @@
-FROM comicrelief/php7-slim:7.1
+FROM crs4/php7-base:7.1
```

Comment:
Excellent! much happier that we depend on our own base if not depending on an official one!